### ISSUE TITLE
Calling set_edge_properties doesn't return error if edge doesn't exist

### ISSUE THREAD

**Issue by ozgrakkurt** [indradb/indradb#192]
There doesn't seem to be any way to know this operation succeeded. Is this really the case?

**Comment by ozgrakkurt** [comment]
So I looked into the code and it seems like It would be easy to convert the api to Result<u64> where the number indicates how many properties were changed. I think this would be useful. Should I do a pr about this maybe?

**Comment by ysimonson** [comment]
Yeah that change makes sense. I don’t know offhand if the various datastore implementations would support returning a count though.

**Comment by ozgrakkurt** [comment]
Working on doing this. I couldn't make much progress because of work but I should have a PR soon.

**Comment by ysimonson** [comment]
@ozgrakkurt I'm going to unassign for now, but if you circle back to this please do let me know
